### PR TITLE
createReservation solved

### PR DESCRIPTION
**Pull request by mihaifirst**

### CODE REVIEW

File: 2024.10.31/01_reservations/r1-alex.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+let id = 0;
```
Comment (vestemeansorin91) [inline 2024.10.31/01_reservations/r1-alex.js]: why you need this ?

I said `id is index+1`

File: 2024.10.31/01_reservations/r1-alex.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    id: id,
+    date: date,
+    categories: [],
+    startHour: startHour,
+    endHour: endHour,
+    range: range,
+  };
+}
+const reservation = createReservation("2024-10-31", 6, 9, 15);
+console.log(reservation);
```
Comment (vestemeansorin91) [inline 2024.10.31/01_reservations/r1-alex.js]: why you left console.log?, code needs to be clean

File: 2024.10.31/01_reservations/r1-alex.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+function createReservation(date, startHour, endHour, range) {
+  id++;
+  return {
+    id: id,
+    date: date,
+    categories: [],
+    startHour: startHour,
+    endHour: endHour,
+    range: range,
+  };
```
Comment (vestemeansorin91) [inline 2024.10.31/01_reservations/r1-alex.js]: shorthand for same key/value pair name
```
 return {
    id,
    date,
    startHour,
    endHour,
    range,
    categories: [],
  };
```

File: 2024.10.31/01_reservations/r1-alex.js (hunk 1 above)
Comment (mihaifirst) [reply 2024.10.31/01_reservations/r1-alex.js]: fixed

**Pull request merged**